doc/conf.py

```diff
@@ -551,3 +551,12 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {"python": ("https://docs.python.org/3/", None)}
+
+
+def setup(app):
+    def skip_doc_element(app, what, name, obj, would_skip, options):
+        if name == "docstring_addendum":
```

Comment:
`if name in autodoc-skip-members:` instead

please create a new variable just after `numpydoc_show_inherited_class_members` named

```python
autodoc-skip-members = [
    "docstring_addendum",
]
```

Reply:
feel free to change this variable name if you have better naming

Reply:
:+1: Done in 0697bf9.